.lando.yml

```diff
@@ -13,9 +13,10 @@ services:
     build:
       - "/app/vendor/bin/phpcs --config-set installed_paths /app/vendor/drupal/coder/coder_sniffer"
     build_as_root:
+      - sed -i 's/DEFAULT@SECLEVEL=2/DEFAULT@SECLEVEL=1/' /etc/ssl/openssl.cnf
```

Comment:
@RichardDavies , do we need this in the PR if we can no longer use lando on the network? I would be okay with removing this from our lando build steps as a quick refactor, but adding changes that don't affect the ability to use lando is probably not necessary. Thoughts?

Reply:
Yeah, this new line should be unnecessary now that I'm outside of the network decryption. I've just removed it but I left the existing cert stuff as I feel like that should be removed as part of a separate story where it can be properly tested.